src/integration-test/java/openapi/OpenApiSpecificationIT.java

```diff
@@ -0,0 +1,82 @@
+package openapi;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import net.javacrumbs.jsonunit.core.Option;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.util.UriComponentsBuilder;
+import se.sundsvall.dept44.util.ResourceUtils;
+import se.sundsvall.precheck.Application;
+
+import java.util.List;
+
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+
+@ActiveProfiles("openapi")
+@SpringBootTest(
+        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        classes = Application.class,
+        properties = {
+                "spring.main.banner-mode=off",
+                "logging.level.se.sundsvall.dept44.payload=OFF"
+        }
+)
+class OpenApiSpecificationIT {
+
+    private static final YAMLMapper YAML_MAPPER = new YAMLMapper();
+
+    @Value("${openapi.name}")
+    private String openApiName;
+    @Value("${openapi.version}")
+    private String openApiVersion;
+
+    @Value("classpath:/openapi.yaml")
+    private Resource openApiResource;
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Test
+    void compareOpenApiSpecifications() {
+        String existingOpenApiSpecification = ResourceUtils.asString(openApiResource);
```

Comment:
You are using `var` in other places. Why mix?

Reply:
The 'String' declaration was an old remnant and a consequence of following the other services too closely.

It is now fixed for consistency. 😓